index.html

```diff
@@ -4,9 +4,16 @@
 
 <head>
 <meta charset="utf-8">
-
+<link rel="stylesheet" href="/main.css"
 <title>Coleman Cox</title>
 </head>
 <body>
+
+
+
+<p>Hello, all.  My name is Coleman Zane Cox and I wanna be somebody, someday.(You can bet your hard-earned dollar I will lol). After roughly 10 hours of Iron Yard class, I can already tell that this is going to take some fight to attain and then surpass my own personal expectations.  Wish me success...
+btw Mr. T,  my css skill is absent atm</p>
+<p>  <img src="http://4.bp.blogspot.com/-IiTLoFgfQrc/TdpwEv4ppBI/AAAAAAAAACI/QKggy7LKZpM/s1600/Yc4Vk.png">
+My social accounts are pretty bare-bones-ed, right now. I belong to Google Plus and git hub. My account is<a href"https://www.youtube.com/watch?v=5SaYQeX1LCE">colez@fff.com</a>
```

Comment:
On your anchor tag (`a`) you have left out the `=` when declaring your `href` attribute.
Since the browser doesn't see this as `href="https://www.youtube.com/watch?v=5SaYQeX1LCE"` it skips over that url entirely and doesn't think there are any attributes on the element: the default behavior for a link without an `href` set properly is to not show styles at all and not link to anything.